apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -474,6 +485,15 @@ struct ComposeView: View {
                 clearState()
             }
         }
+        .onChange(of: composeState.inProgress) { inProgress in
+            if inProgress {
+                DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
```

Comment:
I think 3 is too much, we should change to 1
```suggestion
                DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```